llvm/test/Transforms/LoopVectorize/reduction-with-invariant-store.ll

```diff
@@ -625,10 +625,10 @@ exit:
 define void @reduc_mul_add_store_same_ptr(ptr %dst, ptr readonly %src) {
 ; CHECK-LABEL: define void @reduc_mul_add_store_same_ptr
 ; CHECK:       middle.block:
-; CHECK-NEXT:    [[TMP4:%.*]] = call i32 @llvm.vector.reduce.add.v4i32(<4 x i32> [[TMP3:%.*]])
-; CHECK-NEXT:    store i32 [[TMP4]], ptr %dst, align 4
-; CHECK-NEXT:    [[TMP2:%.*]] = call i32 @llvm.vector.reduce.mul.v4i32(<4 x i32> [[TMP1:%.*]])
-; CHECK-NEXT:    store i32 [[TMP2]], ptr %dst, align 4
```

Comment:
I wonder why this was the correct order of stores

Reply:
...That's a good question. I think this was the incorrect order of stores. 
@fhahn @igogo-x86 Could you help verify it again?

Reply:
New order looks correct, storing the mul before the add, as in the original IR